site/content/en/docs/concepts/cluster_queue.md

```diff
@@ -63,7 +63,7 @@ Kueue assigns the first flavor in the ClusterQueue's `.spec.resourceGroups[*].fl
 list that has enough unused `nominalQuota` quota in the ClusterQueue or the
 ClusterQueue's [cohort](#cohort).
 
-Since `pods` resource name is [reserved](/docs/conceps/workload#reserved-resource-names) and it's value
+Since `pods` resource name is [reserved](/docs/concepts/workload/#reserved-resource-names) and it's value
```

Comment:
Here's the hole, other parts are formats.

Reply:
This is a nice catch.